app/uk/gov/hmrc/customs/emailfrontend/views/email_confirmed.scala.html

```diff
@@ -0,0 +1,35 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.i18n.Lang
+@import partials.main_template
+@import uk.gov.hmrc.customs.emailfrontend.config.AppConfig
+
+
+@this(main_template: main_template)
+@()(implicit request: Request[_], messages: Messages)
+
+@main_template(messages("customs.emailfrontend.email-confirmed.title-and-heading")) {
+
+<div class="column-two-thirds start_page">
+    <h1 class="heading-xlarge">@Messages("customs.emailfrontend.email-confirmed.title-and-heading")</h1>
```

Comment:
shouldn't that be `@messages` rather than `@Messages`? 
same for other two tags below